fe/fe-core/src/main/java/org/apache/doris/qe/ShowExecutor.java

```diff
@@ -1422,7 +1422,7 @@ private void handleShowTablet() throws AnalysisException {
                 }
                 dbName = db.getFullName();
                 Table table = db.getTableNullable(tableId);
-                if (table == null || !(table instanceof OlapTable)) {
```

Comment:
No need to check null?

Reply:
no need, null instanceof always return false